src/app/newcomers/newcomers.component.html

```diff
@@ -1,9 +1,24 @@
 <div class="container-fluid mt-5">
-  <div class=" container box logos mx-auto d-block">
-    <div class="row text-center">
-      <h1 class="mx-auto mb-5"> Welcome to Systers Open Source!</h1>
-      <div class="mx-auto">
+    <div class=" container box logos mx-auto d-block">
+      <div class="row text-center">
+        <h1 class="mx-auto mb-5"> Welcome to Systers Open Source!</h1>
         <p>
+          Systers is an international Anita B.org community encouraging the participation of all women involved in technology.
+        </p>
+        <p>
+          We are committed to providing a friendly, safe, and welcoming environment for all, regardless of gender, sexual orientation,
+          ability, ethnicity, socioeconomic status, work experience, or religion (or lack thereof).
+        </p>
+        <p class="mb-5">
+          We believe in inclusivity, diversity, and transparency. We value open source citizenship and collaborative involvement.
+        </p>
+      </div>
+      <div class="row justify-content-center mb-3">
```

Comment:
I thought you had a previous PR that removed the map from the newcomer's page? Or was it a different page?

Reply:
@bethanyr Done. :)